README.md

```diff
@@ -1,296 +1,93 @@
-# Zowe Explorer
+# Getting Started with Zowe Explorer
 
 [![version](https://vsmarketplacebadge.apphb.com/version-short/Zowe.vscode-extension-for-zowe.png)](https://vsmarketplacebadge.apphb.com/version-short/Zowe.vscode-extension-for-zowe.png)
 [![downloads](https://vsmarketplacebadge.apphb.com/downloads-short/Zowe.vscode-extension-for-zowe.png)](https://vsmarketplacebadge.apphb.com/downloads-short/Zowe.vscode-extension-for-zowe.png)
-[![codecov](https://codecov.io/gh/zowe/vscode-extension-for-zowe/branch/master/graph/badge.svg)](https://codecov.io/gh/zowe/vscode-extension-for-zowe)
-[![slack](https://img.shields.io/badge/chat-on%20Slack-blue)](https://slack.openmainframeproject.org/)
 
-[Zowe Explorer](https://github.com/zowe/community#zowe-explorer) is a sub-project of Zowe, focusing on modernizing mainframe experience. [Zowe](https://www.zowe.org/) is a project hosted by the [Open Mainframe Project](https://www.openmainframeproject.org/), a [Linux Foundation](https://www.linuxfoundation.org/) project.
+Welcome to Zowe Explorer! This project is powered by Zowe, [Open Mainframe Project](https://www.openmainframeproject.org/).
```

Comment:
I prefer the wording: "Zowe, a project hosted by the Open Mainframe Project".

The format "Zowe, Open Mainframe Project" reminds me of the format people use when listing their address (e.g. City, State or City, Country).

Reply:
I think the emphasis here should be put on Zowe Explorer rather than Zowe :) Maybe, something like "Welcome to Zowe Explorer, a project hosted by Open Mainframe Project!"?

Reply:
Sure, that works for me.